### PR TITLE
fix ingest endpoints for data ingestion

### PR DESCRIPTION
**Pull request by jgheewala**

### CODE REVIEW

File: sfxclient/httpsink.go [hunk 1]
```diff
@@ -36,16 +36,16 @@ const (
...
-	IngestEndpointV2 = "https://ingest.signalfx.com/v2/datapoint"
+	IngestEndpointV2 = "https://ingest.us0.signalfx.com/v2/datapoint"
 
 	// EventIngestEndpointV2 is the v2 version of the signalfx event endpoint
-	EventIngestEndpointV2 = "https://ingest.signalfx.com/v2/event"
+	EventIngestEndpointV2 = "https://ingest.us0.signalfx.com/v2/event"
 
 	// TraceIngestEndpointV1 is the v1 version of the signalfx trace endpoint
-	TraceIngestEndpointV1 = "https://ingest.signalfx.com/v1/trace"
+	TraceIngestEndpointV1 = "https://ingest.us0.signalfx.com/v1/trace"
```
Comment (mdubbyap) [inline sfxclient/httpsink.go:45]: this should be at a minimum deprecated, at a maximum removed and major version incremented.

**Review by mdubbyap — COMMENTED** [review]
I also am unsure why this helps anything, it's not like we're going to not be maintaining both...

**Comment by jgheewala** [comment]
> I also am unsure why this helps anything, it's not like we're going to not be maintaining both...

it's to ensure that any customer using defaults are not using old endpoints.

File: sfxclient/httpsink.go (hunk 1 above)
Comment (mdubbyap) [reply sfxclient/httpsink.go:46]: Actually, i've thought more about this. Let's update this to v2/trace saying (leaving the deprecated flag you added) and describe why it's been changed to v2/trace... thoughts?

Comment (jgheewala) [reply sfxclient/httpsink.go]: updated let me know

**Review by mdubbyap — APPROVED** [review]

**Pull request merged**